src/pages/players/players.ts

```diff
@@ -269,6 +270,38 @@ export class PlayersPage {
 		return this.appStorage.getGroups();
 	}
 
+	/**
+	 * Deletes a player from the view and database, after confirmation from user.
+	 *
+	 * @param {Player} player Player to delete.
+	 * @memberof PlayersPage
+	 */
+	private deletePlayer(player: Player): void {
+
+		const confirm = this.alertController.create({
+			title: 'Are you SURE?',
+			message: 'This action cannot be undone. Are you sure you want to delete ' + player.firstName + ' ' +
+					player.lastName + ' from all existence?',
+
+		  buttons: [
+			  {
+				text: 'I guess not...'
+			  },
+			  {
+				text: 'Absolutely!',
+				handler: () => {
+					// Remove deleted player from display of filtered players.
+					this.filteredPlayers.splice(this.filteredPlayers.indexOf(player), 1);
```

Comment:
Should this also remove the deleted player from the list of unfiltered players? Otherwise once the filter is removed then the deleted player would still be in the list.